src/app/modules/main/activity_center/controller.nim

```diff
@@ -38,14 +36,9 @@ method delete*[T](self: Controller[T]) =
   discard
 
 method init*[T](self: Controller[T]) =
-  self.events.on(SIGNAL_ACTIVITY_CENTER_NOTIFICATIONS_LOADED) do(e: Args):
+  self.events.on(activity_center_service.SIGNAL_ACTIVITY_CENTER_NOTIFICATIONS_LOADED) do(e: Args):
     let args = ActivityCenterNotificationsArgs(e)
-    self.delegate.pushActivityCenterNotifications(args.activityCenterNotifications)
-
-  self.events.on(chat_service.SIGNAL_CHAT_UPDATE) do(e: Args):
```

Comment:
@jrainville I am not sure this one was relevant, indeed we are now using the service to handle event. Could you confirm? I am also not seeing any activityCenterNotifications added in SIGNAL_CHAT_UPDATE